docs/userGuide/deployingTheSite.md

```diff
@@ -103,18 +103,23 @@ Since May 2018, Travis CI has been [undergoing migration to `travis-ci.com`](htt
 1. [Add an environment variable in Travis CI](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings) named `GITHUB_TOKEN`, with the value set to the personal access token generated in the previous step. ==Ensure that _Display value in the build log_ is set to _Off_.==
    <include src="screenshot.md" boilerplate var-alt="Add GITHUB_TOKEN" var-file="travisGithubToken.png" inline />
 1. Add a `.travis.yml` file to instruct Travis CI to build and deploy the site when you push to the repository. An example `.travis.yml` file that can accomplish this is given below:
-    ```yaml
-    language: node_js
-    node_js:
-      - '8'
-    install:
-      - npm i -g markbind-cli
-    script: markbind build && markbind deploy --travis
-    branches:
-      only:
-      - master
-    ```
-    More information about `.travis.yml` can be found in the [Travis CI documentation](https://docs.travis-ci.com/).
+```yaml
+language: node_js
+node_js:
+  - 10
+install:
+  - npm i -g markbind-cli
+script: markbind build
+deploy:
+  provider: script
+  script: markbind deploy --travis
+  skip_cleanup: true
+  keep_history: true
```

Comment:
hmm is `keep_history` needed?

Reply:
hmm yeah it doesn't seem necessary since script deployment doesn't do forced pushing (provider `pages`). I'll remove it

>Deploying to GitHub Pages uses git push --force to overwrite the history on the target branch, so make sure you only deploy to a branch used for that specific purpose, such as gh-pages. It is possible to disable this “force push” behavior by setting keep_history option to true.